docs/io/visualization/using_widgets.rst

```diff
@@ -178,4 +178,37 @@ There are also several other options in the modebar which we have not explained
 you remember to click back on the **Box Select** option for making selections on
 spectrum.
 
+Energy Level Widget
+################
+
+This widget lets you visualize the last line interactions
+
+.. image:: ../images/energy_level_widget_demo.gif
+    :alt: Demo of Energy Level Widget
+
+By selecting an ion on the widget, you can see its energy level diagram based
+on the last line interactions in the simulation.
```

Comment:
"...diagram, which also shows information about the last line interactions experienced by packets in the simulation."